portal/app/api/v1/endpoint/farm_data_parser.py

```diff
@@ -77,5 +66,38 @@ def main_power_parser(data):
     farm_df["energy"] = farm_df["POWER_TEST_PEN"] / 12
     # timezone changed from utc to pacific
     farm_df["timestamp"] = utc_to_pst(farm_df["timestamp"])
+    # round df values
+    farm_df = df_round(farm_df)
 
     return farm_df.to_json(date_format="iso")
+
+
+# Helper functions
+
+def celsius_to_fahr(temp_c):
+    """
+        Convert Celsius to Fahrenheit
+    """
+    temp_f = temp_c * 9 / 5 + 32
+
+    return temp_f
+
+
+def utc_to_pst(time):
+    """
+        Convert TimeZone UTC to Pacific
+    """
+    time_utc = pd.to_datetime(time)
+    time_pst = time_utc - datetime.timedelta(hours=8)
+
+    return time_pst
+
+
+def df_round(df):
+    """
+        Rounds the values of a dataframe except the timestamp column
+    """
+    for name in df.columns:
+        if name != "timestamp":
```

Comment:
Can you make this dynamic, or change the name of the function? `df_round` doesn't tell me that it ignores a timestamp column. Maybe call it `df_round(df, columns_to_ignore=['timestamp']):`

Reply:
Would it be appropriate to name the function as it ignores the timestamp column in this case or that it ignores a custom set of columns?